gui/app.py

```diff
@@ -143,6 +143,8 @@ def __init__(self, filenames):
         self.tabs = QTabWidget()
         self.main_tab = MainTab(len(filenames))
         self.debug_tab = DebugTab(len(filenames))
+        self.main_tab.organize()
+        self.debug_tab.organize()
```

Comment:
Calling them separately was the simplest way I could think of to make sure the the constructor for each tab and all of its parent classes was called before any of the organize methods were called. If I put a call to `organize()` at the end of the `RootTab` constructor, it would cause issues when constructing the `DebugTab`. The constructor for `DebugTab` calls `super().__init__()` before adding some of its tab specific widgets, so if that parent constructor called the `organize()` method, DebugTab's organize method would get called before it finished its constructor, and it would try to reference widgets that hadn't been created yet. We could probably work around that by putting all calls to `super().__init__` at the end of the constructor, but I thought this was just easier.